windows/ReactTestApp/MainPage.h

```diff
@@ -1,17 +1,25 @@
 #pragma once
 
 #include "MainPage.g.h"
+#include "ReactInstance.h"
 
 namespace winrt::ReactTestApp::implementation
 {
     struct MainPage : MainPageT<MainPage> {
     public:
         MainPage();
-
-        Windows::Foundation::Collections::IVector<ReactTestApp::ComponentViewModel> Components();
+        void LoadFromJSBundle(Windows::Foundation::IInspectable const &,
+                              Windows::UI::Xaml::RoutedEventArgs);
+        void LoadFromDevServer(Windows::Foundation::IInspectable const &,
+                               Windows::UI::Xaml::RoutedEventArgs);
+        winrt::Windows::Foundation::IAsyncAction
+            OnNavigatedTo(Windows::UI::Xaml::Navigation::NavigationEventArgs);
```

Comment:
Is this an override? Should we add the `override` keyword?
```suggestion
            OnNavigatedTo(Windows::UI::Xaml::Navigation::NavigationEventArgs) override;
```

Reply:
Seems like `OnNavigatedTo` is not declared as `virtual` in base class, so it's not possible to override it.